test/services/notify/notify-status.service.test.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, afterEach, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const { NotifyClient } = require('notifications-node-client')
+const { stubNotify } = require('../../../config/notify.config.js')
+
+// Thing under test
+const NotifyStatusService = require('../../../app/services/notify/notify-status.service.js')
+
+describe('Notify - Status service', () => {
+  let notifyStub
+  let notificationId
+
+  beforeEach(() => {
+    // If you wish to test live notify replace this with a real notification id
+    notificationId = '5a714bec-4ca0-45ba-8edf-8fa37db09499'
+  })
+
+  afterEach(() => {
+    Sinon.restore()
+  })
+
+  describe('when the call to "notify" is successful', () => {
+    beforeEach(() => {
+      notifyStub = _stubSuccessfulNotify()
+    })
+
+    it('should call notify', async () => {
+      const result = await NotifyStatusService.go(notificationId)
+
+      expect(result).to.equal({
+        status: 'received'
```

Comment:
This is a letter status. Should we highlight this or have a email and letter test.